packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -382,6 +382,44 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         return state.pendingDeposits[owner][tokenID].amount;
     }
 
+    function flashDeposit(
+        address to,
```

Comment:
this `to` is not used...

Reply:
I think it is, and it should be. We deposit to some account on L2 so we need an address that will receive the flash deposit and so we process a normal deposit request to this `to` address.